administrator/language/de-DE/com_fields.ini

```
@@ -65,8 +65,8 @@ COM_FIELDS_FIELD_SEARCHINDEX_SEARCHABLE="Durchsuchbar machen"
 COM_FIELDS_FIELD_SEARCHINDEX_TAXONOMY="Als Taxonomie hinzufügen"
 COM_FIELDS_FIELD_SHOWLABEL_DESC="Beschriftung in der Feldausgabe anzeigen oder verbergen."
 COM_FIELDS_FIELD_SHOWLABEL_LABEL="Beschriftung"
-COM_FIELDS_FIELD_SHOWON_DESC="Das Feld kann abhängig vom Wert anderer Felder ein- oder ausgeblendet werden."
-COM_FIELDS_FIELD_SHOWON_LABEL="Attribut anzeigen"
+COM_FIELDS_FIELD_SHOWON_DESC="Dieses Feld in Abhängigkeit von dem Wert(value) eines anderen Feld(field) anzeigen. Eingabe muss folgendem Pattern folgen: field[number]:[value] oder [field-alias]:[value]"
+COM_FIELDS_FIELD_SHOWON_LABEL="Anzeigen in Abhänigkeit"
```

```suggestion
COM_FIELDS_FIELD_SHOWON_DESC="Dieses Feld in Abhängigkeit von dem Wert(value) eines anderen Feldes(field) anzeigen. Die Eingabe muss folgendem Muster folgen: field[number]:[value] oder [field-alias]:[value]"
COM_FIELDS_FIELD_SHOWON_LABEL="Anzeigen in Abhängigkeit"
```
Der Teil mit `field[number]:[value] oder [field-alias]:[value]` wäre doch zuerst im Core anzugehen? Des Weiteren gibt es ja noch die Möglichkeiten `[AND]`, `[OR]` etc. weswegen ich finde, dass die Klammern nicht optimal als Platzhalter geeignet sind. [Form field (Showon)](https://docs.joomla.org/Form_field/de#Showon)

---

Danke @heelc29 
@jduerscheid magst du dann einen Core-PR machen? Bei solchen Änderungen, die ja alle betrifft, sollte dies top-down angegangen werden. Danke.

---

@jduerscheid Wie machen wir hier weiter?